test.js

```diff
@@ -141,17 +141,12 @@ test('Missing header', t => {
     url: '/',
     method: 'GET'
   }, res => {
-    const payload = JSON.parse(res.payload)
-    t.strictEqual(res.statusCode, 400)
-    t.deepEqual({
-      error: 'Bad Request',
-      message: 'Missing `accept encoding` header',
-      statusCode: 400
-    }, payload)
+    t.strictEqual(res.statusCode, 200)
+    t.notOk(res.headers['content-encoding'])
   })
 })
 
-test('Should close the stream', t => {
+test('Should not compress on missing header', t => {
```

Comment:
Can you update this test?
You can pass an invalid header, we want to check that the plugin will close the stream in case of error :)